--- backup.py	2015-04-27 12:30:41.526634426 +0200
+++ driver_other_hook.py	2015-04-28 21:54:52.868920924 +0200
@@ -96,6 +96,7 @@
 from nova.virt import event as virtevent
 from nova.virt import firewall
 from nova.virt import hardware
+from nova.virt.libvirt.hooks import vpci_hook
 from nova.virt.libvirt import blockinfo
 from nova.virt.libvirt import config as vconfig
 from nova.virt.libvirt import dmcrypt
@@ -4299,7 +4300,7 @@
 
         LOG.debug('End _get_guest_xml xml=%(xml)s',
                   {'xml': xml}, instance=instance)
-        return xml
+        return vpci_hook.add_vpci_address_information(self, xml, instance, network_info)
 
     def _lookup_by_id(self, instance_id):
         """Retrieve libvirt domain object given an instance id.
